### ISSUE TITLE
Flush triggered but queue restoration and settings loading not complete. Flush will be retried.

### ISSUE THREAD

**Issue by dhatGuy** [segmentio/analytics-react-native#989]
I upgraded to 2.19.4 and these logs started appearing.
![image](https://github.com/user-attachments/assets/4de515d3-9ded-4cd2-8f85-1e7a92e9dcb9)

I restarted the app and stopped getting these
```
 ERROR  [SyntaxError: JSON Parse error: Unexpected character: <]
 WARN  Possible Unhandled Promise Rejection (id: 0):
 ```
but this keeps getting logged
```
 INFO  Flush triggered but queue restoration and settings loading not complete. Flush will be retried.
 ```

- `analytics-react-native` version: 2.19.4
- Integrations versions (if used): None
- React Native version: 0.74.5
- iOS or Android or both? Android (iOS not tested)

**Steps to reproduce**

**Expected behavior**
No warnings or the same info logged multiple times
**Actual behavior**
Warnings and info logs appearing repeatedly

**Comment by mayfairr** [comment]
Also experiencing this on latest version after upgrading :/

**Comment by itajenglish** [comment]
+1

**Comment by meabed** [comment]
Same here - it would be great to add option to not display this error at all

**Comment by tarmoire** [comment]
+1

**Comment by alanjcharles** [comment]
resolved in #999
